### PR TITLE
Use project fixtures for FileUpdater specs

### PR DESCRIPTION
**Pull request by brrygrdn**
Breaks out from https://github.com/dependabot/dependabot-core/pull/3350

### CODE REVIEW

File: bundler/spec/spec_helper.rb [hunk 1]
```diff
@@ -27,6 +27,10 @@ def bundler_project_dependency_file(project, filename:)
   dependency_file
 end
 
+def bundler_build_tmp_repo(project)
+  build_tmp_repo(project, path: "projects/bundler1")
```
Comment (brrygrdn) [inline bundler/spec/spec_helper.rb:31]: This is hard-coded to bundler1, it'll switch based on the suite name in #3350

File: bundler/spec/dependabot/bundler/file_updater_spec.rb [hunk 2]
```diff
@@ -27,25 +27,7 @@
...
-    Dependabot::DependencyFile.new(
-      name: "Gemfile.lock",
-      content: lockfile_body,
-      directory: directory
-    )
-  end
-  let(:gemfile_body) { fixture("ruby", "gemfiles", gemfile_fixture_name) }
-  let(:lockfile_body) { fixture("ruby", "lockfiles", lockfile_fixture_name) }
-  let(:gemfile_fixture_name) { "Gemfile" }
-  let(:lockfile_fixture_name) { "Gemfile.lock" }
```
Comment (feelepxyz) [inline bundler/spec/dependabot/bundler/file_updater_spec.rb:48]: 💥

**Review by feelepxyz — APPROVED** [review]
LGTM 👍

**Comment by feelepxyz** [comment]
🥳

**Pull request merged**